docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/signals/istft-16.rst

```diff
@@ -0,0 +1,206 @@
+* **3**: ``frame_size`` - Scalar tensor of type *T_INT* describing the size of a single frame of the signal to be provided as input to FFT. **Required.**
+* **4**: ``frame_step`` - Scalar tensor of type *T_INT* describing the distance (number of samples) between successive frames. **Required.**
+* **5**: ``signal_length`` - Scalar or single element 1D tensor of type *T_INT* describing the desired length of the output signal, if not provided it's calculated accordingly to the rules presented in the detailed description above. **Optional.**
+
+
+**Outputs**
+* **1**: ``signal`` - Tensor of type *T* and 1D shape ``[signal_length]`` or 2D shape ``[batch, signal_length]`` with a real valued signal data. **Required.**
+
+**Types**
+
+* *T*: any supported floating-point type.
+
+* *T_INT*: ``int64`` or ``int32``.
+
+
+**Examples**:
+
+*Example 3D input, 1D output signal, center=false, default signal_length:*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="ISTFT" ... >
+        <data center="false" ... />
+        <input>
+            <port id="0">
+                <dim>6</dim>
+                <dim>16</dim>
+                <dim>2</dim>
+            </port>
+            <port id="1">
+                <dim>7</dim>
+            </port>
+            <port id="2"></port> <!-- frame_size value: 11 -->
+            <port id="3"></port> <!-- frame_step value: 3 -->
+        <output>
+            <port id="4">
+                <dim>56</dim>
+            </port>
+        </output>
+    </layer>
+
+*Example 4D input, 2D output signal, center=false, default signal_length:*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="ISTFT" ... >
+        <data center="false" ... />
+        <input>
+            <port id="0">
+                <dim>4</dim>
+                <dim>6</dim>
+                <dim>16</dim>
+                <dim>2</dim>
+            </port>
+            <port id="1">
+                <dim>7</dim>
+            </port>
+            <port id="2"></port> <!-- frame_size value: 11 -->
+            <port id="3"></port> <!-- frame_step value: 3 -->
+        <output>
+            <port id="4">
+                <dim>4</dim>
+                <dim>56</dim>
+            </port>
+        </output>
+    </layer>
+
+
+*Example 3D input, 1D output signal, center=true, default signal_length:*
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="ISTFT" ... >
+        <data center="true" ... />
+        <input>
+            <port id="0">
+                <dim>6</dim>
+                <dim>16</dim>
+                <dim>2</dim>
+            </port>
+            <port id="1">
+                <dim>7</dim>
+            </port>
+            <port id="2"></port> <!-- frame_size value: 11 -->
+            <port id="3"></port> <!-- frame_step value: 3 -->
```

Comment:
Sometimes the two comments are swapped, in the Example below it's `frame_step` and `frame_size`.